File: llm_generated_code/ChatGPT/Python/Level1/Q31.py
def game_winner(a, b):
    return "Alice" if min(a, b) % 2 == 1 else "Bob"

t = int(input())
results = []
for _ in range(t):
    a, b = map(int, input().split())
    results.append(game_winner(a, b))
print("\\n".join(results))

File: llm_generated_code/ChatGPT/CPP/Level1/Q31.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--) {
        int a, b;
        cin >> a >> b;
        if (abs(a - b) <= 1) {
            cout << "Alice" << endl;
        } else {
            cout << "Bob" << endl;
        }
    }
    return 0;
}
